test/libsolidity/syntaxTests/nameAndTypeResolution/206_storage_location_local_variables.sol

```diff
@@ -1,11 +1,10 @@
 contract C {
-    function f() public {
-        uint[] storage x;
+    uint[] m_x;
+    function f() public view {
+        uint[] storage x = m_x;
         uint[] memory y;
         uint[] memory z;
```

Comment:
Could this line be removed? It seems to be testing the same than the line before.